filesystems/gcs/src/main/scala/cromwell/filesystems/gcs/GcsEnhancedRequest.scala

```diff
@@ -1,13 +1,13 @@
 package cromwell.filesystems.gcs
 
-import java.io.FileNotFoundException
-
 import akka.http.scaladsl.model.StatusCodes
 import cats.effect.IO
 import com.google.api.client.googleapis.json.GoogleJsonResponseException
 import com.google.cloud.storage.StorageException
 import cromwell.filesystems.gcs.RequesterPaysErrors.isProjectNotProvidedError
 
+import java.io.FileNotFoundException
```

Comment:
Not sure where this is used... this file seems unchanged except for whitespace changes?

Reply:
Yes that's right. IntelliJ fixed up some trailing whitespace and non-canonical import ordering when I was in this file figuring out the root cause.